app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="ninja.knowone.apod">
 
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```

Comment:
If you plan on making this the last PR before release, remember to roll your `versionName` and your `versionCode`. Also, I'd wait to merge until the .apk or .aab is built and published so that if there are any last second tweaks you can add them without multiple merges to master. I'll post a checklist to your Trello board.